### DISCUSSION TITLE
Switching to another PC, would like to easily migrate my ferium profiles.

### DISCUSSION THREAD

**Discussion by kaz-49** [gorilla-devs/ferium#261] [Q&A] [answered]
Is there like a .json file or something that I could pull out and stick into the other computer? I have a huge list of mods and I'd really rather not type in every single one again.

**Comment by JustSimplyKyle** [comment, marked as answer]
Yes!
> Ferium stores profile and modpack information in its config file. By default it is located at ~/.config/ferium/config.json, but you can change this in 2 ways. You can set the FERIUM_CONFIG_FILE environment variable or set the --config-file global command flag, the flag always takes precedence.

If you're using windows, it's probably just located in your user folder, I'm not sure tho

**Reply by kaz-49** [reply]
Thank you! It worked! 

The file was at `C:\Users\username\.config\ferium` for anyone else who stumbles across this thread.

**Reply by galeofperil** [reply]
@Kaz-49 this should be its own paragraph in the readme imo

**Reply by theRookieCoder** [reply]
This paragraph is from the readme by the way
https://github.com/gorilla-devs/ferium/blob/a3362882cc4e73c2ea0850a2fa296d7120642473/README.md?plain=1#L119

**Reply by galeofperil** [reply]
@theRookieCoder yeah, but there's no "Migrating Profiles" subheader so it's easily missable
